README.md

```diff
@@ -1,2 +1,64 @@
-# tg5
-TC39-TG5: Experiments in programming language standardization
+# TC39-TG5: Experiments in programming language standardization
+
+This repository contains documents, agendas, and notes for the [ECMAScript - Research Task Group (TC39-TG5)](https://ecma-international.org/task-groups/tc39-tg5/).
+
+See [Scope](#scope) for our mandate.
+
+## Agenda
+
+To be posted in /agendas
+
+## Meetings
+
+Meetings occur the last Wednesday of the month and appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
+
+Meeting link: https://meet.google.com/oti-hukx-zgc
+
+## Notes
+
+To be posted in /meetings
+
+### Last Wednesday of Every Month (Alternating)
+
+Timeslot A
+
+|              |                 |
+| -----------: | --------------- |
+| US / Central | 10:00 Wednesday |
+|          CET | 16:00 Wednesday |
+|        China | 23:00 Wednesday  |
+
+Timeslot B
+
+|              |                 |
+| -----------: | --------------- |
+| US / Central | 04:00 Wednesday |
+|          CET | 10:00 Wednesday |
+|        China | 17:00 Wednesday  |
+
```

Comment:
we could also add
```
| Seoul	| 18:00 Wednesday |
| Tokyo	| 18:00 Wednesday |
```

Reply:
I also noticed that we might want to do the alternating time at just 2 hours earlier -- USA would still be able to join, it would be a little early though. The current a/b slot is maybe not ideal. We can discuss in an issue.

Reply:
you could go crazy like node does too:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 19-Mar-2024 10:00 (10:00 AM) |
| US / Mountain | Tue 19-Mar-2024 11:00 (11:00 AM) |
| US / Central  | Tue 19-Mar-2024 12:00 (12:00 PM) |
| US / Eastern  | Tue 19-Mar-2024 13:00 (01:00 PM) |
| EU / Western  | Tue 19-Mar-2024 17:00 (05:00 PM) |
| EU / Central  | Tue 19-Mar-2024 18:00 (06:00 PM) |
| EU / Eastern  | Tue 19-Mar-2024 19:00 (07:00 PM) |
| Moscow        | Tue 19-Mar-2024 20:00 (08:00 PM) |
| Chennai       | Tue 19-Mar-2024 22:30 (10:30 PM) |
| Hangzhou      | Wed 20-Mar-2024 01:00 (01:00 AM) |
| Tokyo         | Wed 20-Mar-2024 02:00 (02:00 AM) |
| Sydney        | Wed 20-Mar-2024 04:00 (04:00 AM) |

😅

Reply:
my brain is too fried right now to get this right. but we can discuss it and get a good solution for this. we also need to discuss the website at some point and maybe there we can have meeting time localized.